trusted_os/rpmb.go

```diff
@@ -55,11 +58,10 @@ const (
 )
 
 type RPMB struct {
-	storage   *usdhc.USDHC
 	partition *rpmb.RPMB
 }
 
-func (r *RPMB) init() (err error) {
+func newRPMB(storage Card) (r *RPMB, err error) {
```

Comment:
Prefer `new` to avoid repetitive name.

https://google.github.io/styleguide/go/decisions#package-vs-exported-symbol-name

Reply:
I forgot about this, thanks! However, this is part of the `main` package, so I don't think it'd be very clear if we just called `rpmb := new()` in `main.go`. If we decide to move this to its own package, I think we should definitely rename it to just `new`. What do you think?

Reply:
You are right. I didn't notice this is part of the `main` package. `newRPMB` sounds good to me then.